terraform/environments/bootstrap/delegate-access/iam.tf

```diff
@@ -179,6 +179,166 @@ resource "aws_iam_policy" "member-access" {
   policy      = data.aws_iam_policy_document.member-access.json
 }
 
+module "instance-scheduler-access" {
+  count  = local.account_data.account-type == "member" && terraform.workspace != "testing-test" ? 1 : 0
+  source = "github.com/ministryofjustice/modernisation-platform-terraform-cross-account-access?ref=v2.2.0"
+  providers = {
+    aws = aws.workspace
+  }
+  account_id             = local.modernisation_platform_account.id
+  additional_trust_roles = [format("arn:aws:iam::%s:role/github-actions", local.environment_management.account_ids[terraform.workspace])]
+  policy_arn             = aws_iam_policy.instance-scheduler-access[0].id
+  role_name              = "InstanceSchedulerAccess"
+}
+
+#tfsec:ignore:aws-iam-no-policy-wildcards
+data "aws_iam_policy_document" "instance-scheduler-access" {
+  statement {
+    #checkov:skip=CKV_AWS_108
+    #checkov:skip=CKV_AWS_111
+    #checkov:skip=CKV_AWS_107
+    #checkov:skip=CKV_AWS_109
+    #checkov:skip=CKV_AWS_110
+    effect = "Allow"
+    actions = [
+      "ec2:Describe*",
+      "ec2:*SecurityGroup*",
+      "ec2:*KeyPair*",
+      "ec2:*Tags*",
+      "ec2:*Volume*",
+      "ec2:*Snapshot*",
+      "ec2:*Ebs*",
+      "ec2:*NetworkInterface*",
+      "ec2:*Address*",
+      "ec2:*Image*",
+      "ec2:*Event*",
+      "ec2:*Instance*",
+      "ec2:*CapacityReservation*",
+      "ec2:*Fleet*",
+      "ec2:Get*",
+      "ec2:SendDiagnosticInterrupt",
+      "ec2:*LaunchTemplate*",
+      "ec2:*PlacementGroup*",
+      "ec2:*IdFormat*",
+      "ec2:*Spot*"
+    ]
+    resources = ["*"] #tfsec:ignore:AWS099 tfsec:ignore:AWS097
+  }
+
+  statement {
+    effect = "Allow"
+    actions = [
+      "sts:AssumeRole"
+    ]
+    principals {
+      type        = "Service"
+      identifiers = ["lambda.amazonaws.com"]
+    }
+  }
+
+  statement {
+    effect = "Allow"
+    actions = [
+      "sts:AssumeRole"
+    ]
+    resources = [
+      "arn:aws:iam::${local.modernisation_platform_account.id}:role/developer",
+      "arn:aws:iam::${local.environment_management.account_ids[terraform.workspace]}:role/developer"
+    ]
+  }
+
+  statement {
+    sid    = "AllowLambdaToCreateLogGroup"
+    effect = "Allow"
+    actions = [
+      "logs:CreateLogGroup"
+    ]
+    resources = [
+      format("arn:aws:logs:eu-west-2:%s:*", local.modernisation_platform_account.id),
+      format("arn:aws:logs:eu-west-2:%s:*", local.environment_management.account_ids[terraform.workspace])
+    ]
+  }
+
+  statement {
+    sid    = "AllowLambdaToWriteLogsToGroup"
+    effect = "Allow"
+    actions = [
+      "logs:CreateLogStream",
+      "logs:PutLogEvents"
+    ]
+    resources = [
+      format("arn:aws:logs:eu-west-2:%s:*", local.modernisation_platform_account.id),
+      format("arn:aws:logs:eu-west-2:%s:*", local.environment_management.account_ids[terraform.workspace])
+    ]
+  }
+
+  statement {
```

Comment:
This and the following statement look to be a copy and past of the original role and are not needed here